src/main/java/ee/tuleva/onboarding/epis/EpisService.java

```diff
@@ -76,7 +91,7 @@ public CashFlowStatement getCashFlowStatement(Person person, LocalDate fromDate,
     }
 
     @Caching(evict = {
-        @CacheEvict(value = TRANSFER_APPLICATIONS_CACHE_NAME, key = "#person.personalCode"),
+        @CacheEvict(value = APPLICATIONS_CACHE_NAME, key = "#person.personalCode"),
```

Comment:
You should evict both `TRANSFER_APPLICATIONS_CACHE_NAME` and `APPLICATIONS_CACHE_NAME`